script.js

```diff
@@ -0,0 +1,76 @@
+let booksArr = [];
+
+function removeBook(bookIndex) {
+  booksArr = booksArr.filter((book) => book !== booksArr[bookIndex]);
+}
+
+function addBook(bookTitle, bookAuthor) {
+  const book = {
+    title: bookTitle,
+    author: bookAuthor,
+  };
+
+  booksArr.push(book);
+}
+
+function addAwesomeBooksToLocalStorage() {
+  const booksArrString = JSON.stringify(booksArr);
+  localStorage.setItem('awesomeBooks', booksArrString);
+}
+
+function showBooks() {
+  const booksDiv = document.querySelector('.books');
+  booksDiv.innerHTML = '';
+  for (let i = 0; i < booksArr.length; i += 1) {
+    const bookDiv = document.createElement('div');
+    bookDiv.classList.add('book');
+    booksDiv.appendChild(bookDiv);
+
+    const titleParagraph = document.createElement('p');
+    titleParagraph.classList.add('title');
+    titleParagraph.textContent = booksArr[i].title;
```

Comment:
- [ ] I would recommend using `ES6` syntax instead of the `function` keyword. to make the code cleaner and simpler.